JobConfig/mixing/LowIntensity.fcl

```diff
@@ -0,0 +1,8 @@
+#
+# Low intensity beam configuration
+#
+# original author: Dave Brown, LBNL
+#
+physics.producers.PBISim.SDF: 0.6
+physics.producers.PBISim.extendedMean: 1.58e5 #  1% of a single booster batch
```

Comment:
A small clarification on the comment.  This is the 1% of one pulse from the single booster batch configuration, not 1% of a single booster batch.  So maybe "1% of the uncut distribution from a single booster batch".

Reply:
done